locales/sv/validation.php

```diff
@@ -90,17 +90,17 @@
         'numeric' => ':Attribute måste vara större än :min.',
         'string'  => ':Attribute måste innehålla minst :min tecken.',
     ],
-    'multiple_of'          => 'The :attribute must be a multiple of :value',
+    'multiple_of'          => ':attribute måste vara en multipel av :value',
     'not_in'               => ':Attribute är ogiltigt.',
     'not_regex'            => 'Formatet för :attribute är ogiltigt.',
     'numeric'              => ':Attribute måste vara en siffra.',
     'password'             => 'Lösenordet är fel.',
     'present'              => ':Attribute måste finnas med.',
-    'prohibited'           => 'The :attribute field is prohibited.',
-    'prohibited_if'        => 'The :attribute field is prohibited when :other is :value.',
-    'prohibited_unless'    => 'The :attribute field is prohibited unless :other is in :values.',
+    'prohibited'           => 'Fältet :attribute är förbjudet.',
+    'prohibited_if'        => ':attribute-fältet är förbjudet när :other är :value.',
```

Comment:
Would this render weirdly as it's a variable connected with a dash? It's the correct way, but I think it would make sense just saying ":attribute är förbjudet.." (just removing "-fältet")